Dockerfile

```diff
@@ -9,7 +9,12 @@ ARG MB_EDITION=oss
 WORKDIR /app/source
 
 COPY . .
-RUN NODE_ENV=production MB_EDITION=$MB_EDITION yarn --frozen-lockfile && yarn build && bin/i18n/build-translation-resources
+
+RUN yarn --frozen-lockfile
+
+ENV NODE_ENV=production
+
+RUN MB_EDITION=$MB_EDITION yarn build && bin/i18n/build-translation-resources
```

Comment:
While this works, I think a simpler alternative will be (note the newly added  `production` flag to yarn, per [its documentation](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-production-true-false)):

```
COPY . .
RUN NODE_ENV=production MB_EDITION=$MB_EDITION yarn --production=false --frozen-lockfile && yarn build && bin/i18n/build-translation-resources
```